lib/active_admin/formatter/base.rb

```diff
@@ -0,0 +1,19 @@
+module ActiveAdmin
+  module Formatter
+    class Base
+      extend ActiveAdmin::Formatter::DSL
+
+      def initialize(object, context)
+        @object, @context = object, context
+      end
+
+      detection do |object, context|
+        true
+      end
+
+      formater do |object, context|
```

Comment:
Unused block argument - `context`. If it's necessary, use `_` or `_context` as an argument name to indicate that it won't be used.